Code/Framework/AzToolsFramework/AzToolsFramework/Archive/ArchiveComponent.cpp

```diff
@@ -132,7 +112,7 @@ namespace AzToolsFramework
         {
             serializeContext->Class<ArchiveComponent, AZ::Component>()
                 ->Version(2)
```

Comment:
this seems like a significant enough refactor to update the version of this component

Reply:
Serialization wise, nothing has changed to the SystemComponent.

Plus we don't actually serialize out the SystemComponent anymore anyway as we have gotten rid of the Editor.xml/Game.xml.

Really we expect SystemComponents to not use the Serialization system for it's settings, but instead use the SettingsRegistry to query it's settings from.

Reply:
I am arguing to not change the serialization version of any of the SystemComponents anymore and just leave them as is.

Reply:
Will do.

Reply:
> I am arguing to not change the serialization version of any of the SystemComponents anymore and just leave them as is.

These versions aren't just used for serialized content. They are used in other ways, for example the fingerprint for the scene processing Gem is a composite of the versions of several components.

The serialization version is the only version available for many classes, so it's really useful to bump it up when logic changes in a system, even if the actual data hasn't changed. Otherwise there isn't a way to track when this occurs in external code.

Reply:
This isn't a regular Game component though. This a System Component whose outward functionality remains the same.
The Archive ABI is compatible with any archive files made before the changes.

Builders don't need to re-run because of this change.